src/helpers/Sign/ethsecp256k1hdwallet.js

```diff
@@ -0,0 +1,178 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
is there any pre-existing library we can use here instead of having the code explicitly implemented?

Reply:
That's a great idea. Currently, there are no implementations that are fully compatible with cosmjs and eth signatures, so we need to adapt the signature methods here. 

But the ethsecp256k1hdwallet signature uses the libraries of web3 and ethers, And I have removed the redundant dependency encryption libraries. Thank you for your advice